api/repositories/service_binding_repository_test.go

```diff
@@ -1149,6 +1155,174 @@ var _ = Describe("ServiceBindingRepo", func() {
 		})
 	})
 
+	Describe("GetServiceBindingParameters", func() {
+		var (
+			serviceBindingGUID   string
+			searchGUID           string
+			serviceBindingParams repositories.ServiceBindingParametersRecord
+			getErr               error
+			serviceInstance      *korifiv1alpha1.CFServiceInstance
```

Comment:
Those four variables (`serviceInstance`, `servicePlan`, `serviceBroker` and `offering`) are only used locally in `BeforeEach`. Therefore they do not need to be declared here